package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "description": "Sudoku is a logic-based, combinatorial number-placement puzzle.",
   "author": "Mark <mark@remarkablemark.org>",
+  "engines": {
```

Comment:
Just a minor nitpick, do you mind moving `engines` field to above the `private` field?